pkg/alertdog/alertdog.go

```diff
@@ -26,7 +26,7 @@ type Pagerduty interface {
 type Alertdog struct {
 	AlertmanagerEndpoints []string `yaml:"alertmanager_endpoints"`
 	Expected              []*Prometheus
-	CheckInterval         time.Duration
+	CheckInterval         time.Duration `yaml:"check_interval"`
 	Expiry                time.Duration
 	Port                  uint
```

Comment:
```suggestion
	Expiry                time.Duration `yaml:"expiry"`
	Port                  uint `yaml:"port"`
```

Reply:
This isn't required if the only difference between the name of the struct field and the yaml name is the capitalisation.  See https://github.com/go-yaml/yaml

This makes sense, because you can only deserialise yaml to exported fields and yaml config often has lowercase names.